.github/workflows/build.yml

```diff
@@ -0,0 +1,137 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+    paths-ignore:
+      - 'bld/bin/**'
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - 'bld/bin/**'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: "cb"
+    - uses: actions/checkout@v2
+      with:
+        repository: libtom/libtomcrypt
+        path: "libtomcrypt"
+        ref: "5a3a12c9b343e219a0a1cbc3c47e997e5a39ffa6"
```

Comment:
I followed the `readme` wrt. the libtomcrypt version but it should probably be switched to official release tag.

Reply:
Agreed.